packages/xo-server/bin/xo-server

```diff
@@ -12,16 +12,18 @@ if (process.env.NODE_ENV === undefined) {
 // Better stack traces if possible.
 try {
   require('source-map-support').install({
-    handleUncaughtExceptions: false
-  });
+    handleUncaughtExceptions: false,
+  })
 } catch (_) {}
 
 // Use Bluebird for all promises as it provides better performance and
 // less memory usage.
 global.Promise = require('bluebird')
 
-require('@xen-orchestra/log/configure').catchGlobalErrors(
-  require('@xen-orchestra/log').default('xo:xo-server')
-)
+// Enable `async_hooks` because it's used by `node-zone`
+// See: http://bluebirdjs.com/docs/api/promise.config.html#async-hooks
+Promise.config({ asyncHooks: true })
```

Comment:
`Bluebird.config`.

Reply:
```js
Promise = Bluebird

Which means

Promise.config === Bluebird.config
```

Reply:
Yes, but this is Bluebird's config, not a standard config for any promise implementations, and, IMHO, for clarity it should be written as `Bluebird.config` in the code.

Reply:
It's understandable on reading the code and it's the same as the bluebird example http://bluebirdjs.com/docs/api/promise.config.html

Reply:
Please make the change.

Reply:
```suggestion
Bluebird.config({ asyncHooks: true })
```